README.md

```diff
@@ -17,3 +17,7 @@ For information about contributing and developing the website locally, please re
 ### Spectre.css
 
 The MEI website uses [Spectre.css](https://picturepan2.github.io/spectre/) as published at <https://github.com/picturepan2/spectre> under the terms of the MIT-license.
+
+## Continuous integration and deployment
+
+This website is being built automatially whenever a commit is pushed to the main branch. The build is being executed through our custom github action instead of github-pages. This decision was made due to the limitations githup-pages poses in terms of available jekyll plugins (gems). Nevertheless, the technology for the website is basically the same as when using github-pages, which is still being used for deploying the website.
```

Comment:
```suggestion
This website is being built automatially whenever a commit is pushed to the main branch. The build is being executed through our custom GitHub Action instead of `github-pages`. This decision was made due to the limitations github-pages poses in terms of available jekyll plugins (gems). Nevertheless, the technology for the website is basically the same as when using github-pages, which is still being used for deploying the website.
```